src/ui/components/results/verticalresultscomponent.js

```diff
@@ -191,14 +206,14 @@ export default class VerticalResultsComponent extends Component {
     this.resultsCount = data.resultsCount;
     this.verticalKey = data.verticalConfigId;
     this.resultsContext = data.resultsContext;
-    this.appliedQueryFilters = (data.appliedQueryFilters || [])
+    this.nlpFilters = (data.appliedQueryFilters || [])
       .filter(f => !this._config.hiddenFields.includes(f.fieldId));
     const searchState = data.searchState || SearchStates.PRE_SEARCH;
     const displayResultsIfExist = this._config.isUniversal ||
       this._displayAllResults ||
       data.resultsContext === ResultsContext.NORMAL;
     const showResultsHeader = this.resultsHeaderOpts.showResultCount ||
-      (this.resultsHeaderOpts.showAppliedFilters && this.appliedQueryFilters.length > 0);
+      (this.resultsHeaderOpts.showAppliedFilters && this.nlpFilters.length > 0);
```

Comment:
Does this logic need to change? We're only going to show applied filters if nlpFilters are present. What if there are no nlpFilters but there are static filters and facets? Is that possible?

Reply:
yup updated!